### DISCUSSION TITLE
Can't connect to steam

### DISCUSSION THREAD

**Discussion by vQass** [SteamRE/SteamKit#1445] [General]
Hi, I can't connect to steam using steamClient.Connect.

I enabled logging and there is some socket exception 

ServerList: Next server candidate: 162.254.195.71:443 (Tcp)
288c2b9e71714be5983f3d3fd7ce20dd/TcpConnection: Connecting to 162.254.195.71:443...
288c2b9e71714be5983f3d3fd7ce20dd/TcpConnection: Connected to 162.254.195.71:443
288c2b9e71714be5983f3d3fd7ce20dd/TcpConnection: Socket exception occurred while reading packet: System.IO.IOException: Connection lost while reading packet header.
 ---> System.IO.EndOfStreamException: Unable to read beyond the end of the stream.
   at System.IO.Stream.ReadAtLeastCore(Span`1 buffer, Int32 minimumBytes, Boolean throwOnEndOfStream)
   at System.IO.BinaryReader.InternalRead(Int32 numBytes)
   at System.IO.BinaryReader.ReadUInt32()
   at SteamKit2.TcpConnection.ReadPacket()
   --- End of inner exception stack trace ---
   at SteamKit2.TcpConnection.ReadPacket()
   at SteamKit2.TcpConnection.NetLoop()
   
   Does anyone know what could cause the problem?
   
   I tried windows 10 and 11, nothing helped.

**Comment by LossyDragon** [comment]
Something is going on with the CM servers for a while today. It's hit and miss on what server is actually working or not. 

I've had success with servers on cellid 76, with others having success on Chile and Japan servers. I don't have a list so you'll just have to keep trying or wait it out.

**Reply by vQass** [reply]
Oh, okay. I will try to find working one. Thanks for the advice!
